zephyr/include/sof/init.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2016,2024 Intel Corporation.
+ */
+
+#ifndef __SOF_INIT_H__
+#define __SOF_INIT_H__
+
+struct sof;
```

Comment:
nitpick - this can go after line 16 under `#if CONFIG_MULTICORE`

Reply:
Ack @lyakh , uploaded V2 with this fixed.